packages/components/src/Chip/Chip.tsx

```diff
@@ -44,17 +48,25 @@ export const Chip = ({
 
   return (
     <Tooltip message={tooltipMessage}>
-      <button
+      <Tag
         className={classes}
-        onClick={(ev: React.MouseEvent<HTMLButtonElement>) =>
-          onClick && onClick(value, ev)
+        onClick={(ev: React.MouseEvent<HTMLButtonElement | HTMLDivElement>) =>
+          onClick &&
+          !actAsFormElement &&
```

Comment:
this is interesting tho as we're swallowing the `onClick` when you have `actAsFormElement` as true. A bit of a clashing prop.

why not just skip the extra prop and define the tag logic from onClick existing?